plugins/health.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: health
+spec:
+  homepage: https://github.com/iNecas/kube-health
+  shortDescription: An easy way to determine the health of Kubernetes resources
```

Comment:
Try to shorten as much as you can. We truncate at 55 chars while displaying this. Phrases like "An easy way to" are often redundant.

Reply:
Updated